mne/viz/evoked.py

```diff
@@ -126,6 +127,36 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos=None, colors=None, axis=None, outlines='head'):
```

Comment:
I don't think you need to have default values here since you don't have checks for them.